operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1937,6 +1937,9 @@ metaMonitoring:
           labelSelectors:
             app.kubernetes.io/name: kube-state-metrics
 
+      # -- The scrape interval for all ServiceMonitors.
+      scrapeInterval: 60s
```

Comment:
Changelog says this should be 30s ?

Reply:
I decided to leave it as 60s but forgot to update the changelog too